roles/common/templates/networkpolicy.yml.j2

```diff
@@ -0,0 +1,14 @@
+kind: NetworkPolicy
```

Comment:
should we name this uperf-networkpolicy, or better yet, should we template the workload name?

Reply:
That is the kind which is `NetworkPolicy`..the name as you can see below is 

"{{ meta.name }}-networkpolicy-{{ trunc_uuid }}"

Am I missing something?